app/assets/stylesheets/application.scss

```diff
@@ -140,13 +137,13 @@ main {
   }
 
   .breadcrumb-trail {
-    @include core-16;
-    margin: 10px 0 0 0;
-    padding: 0 0 10px;
-    border-bottom: 1px solid $border-colour;
+    @include govuk-font(16);
+    margin: govuk-spacing(2) 0 0 0;
+    padding: 0 0 govuk-spacing(2);
+    border-bottom: 1px solid $govuk-border-colour;
 
     strong {
-      font-weight: bold;
+      @include govuk-typography-weight-bold;
```

Comment:
Just curious - what does this mixin provide over the standard `font-weight: bold`?

Reply:
actually we can just use 700 which is what above mixin pulls in (`$govuk-font-weight-bold`)

Reply:
I'd definitely encourage people to use the mixin (or `govuk-font` with the `$weight` attribute) so that A) if we ever change the weight of the bold font (admittedly unlikely) it's easy to roll out but also B) if we ever introduce any additional complexity that depends on weight (like `font-size-adjust`, which was [dependent on font weight in frontend toolkit](https://github.com/alphagov/govuk_frontend_toolkit/blob/03204449dcbf28c2d48e7f6dd217f6abe9b3a518/stylesheets/_typography.scss#L31-L35) we can make sure those adjustments get applied auto-magically.